### DISCUSSION TITLE
Adding a help button to Neurodesktop menu

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#125] [Ideas]
I suggest to add a help button to Neurodesktop menu, which will link to the neurodesktop help page in our documentation website (https://neurodesk.github.io/docs/neurodesktop/).
Something like Windows has. I know that our website is the homepage when opening the browser, but people might change the homepage (or never use it -- after all, it is often much more comfortable to use the browser on the host machine).

**Comment by stebo85** [comment]
Dear @civier 

Yes, this is possible. If you can send us a pull request that does this, we can add it easily.

Kind regards
Steffen

**Comment by civier** [comment]
Hi @stebo85

Sound good. I will work on it when I have time.
Another question -- some of the information I want it to present is the version of Neurodesktop being ran, but I'll need some help with that:
1) Where can I take the version number from?
2) Are we planning to have a version-dependent help site? MRtrix and fMRIprep style? If yes, I can use the version number to open the correct documentation page.

I also consider having separate "Help" and "About" buttons, as for version number, people usually look for an "About" button. The "about" button can also link to the webpage with the acknowledgments.

Best,
Oren

**Reply by aswinnarayanan** [reply]
Hi @civier 
Site versioning is a functionality that is available in hugo docsy, but it is currently disabled for the neurodesk.github.io site.
If the site is not versioned, you don't need the neurodesktop version correct?

The production version that the docs reference is contained in this file https://github.com/NeuroDesk/neurodesk.github.io/blob/hugo-docsy/data/neurodesktop.toml
